designer/server/src/test/scala/pl/touk/nussknacker/ui/statistics/StatisticsSpec.scala

```diff
@@ -52,4 +53,22 @@ class StatisticsSpec extends AnyFunSuite with Matchers {
     Statistics.toURL("xd://stats.nussknacker.io/?f=a+b") shouldBe Left(CannotGenerateStatisticsError)
   }
 
+  test("should return encrypted URL if not set otherwise") {
```

Comment:
I would also test length limit of encrypt URL, what I mean is to set query params to actual URL limit and then test with encryption